core/database.py

```
@@ -182,7 +175,10 @@ async def delete_message(message_id: int) -> bool:
                 message_id,
             )
             deleted_count = int(result.split()[-1]) if result else 0
-            logger.debug(f"Deleted message {message_id} from database")
+            if deleted_count > 0:
+                logger.debug(f"Deleted message {message_id} from database")
```

Ruff (0.15.2)

[warning] Logging statement uses f-string

(G004)